components/feature/accounts-push/src/main/java/mozilla/components/feature/accounts/push/SendTabUseCases.kt

```diff
@@ -174,3 +182,11 @@ internal inline fun filterSendTabDevices(
         }
     }
 }
+
+@VisibleForTesting
+internal fun isValidTabSchema(tab: TabData): Boolean {
+    // We don't sync certain schemas, about|resource|chrome|file|blob|moz-extension
+    // See https://searchfox.org/mozilla-central/source/modules/libpref/init/all.js#4372
```

Comment:
nit: This line you meant to point at has now moved. In searchfox, if you highlight a line and press 'y' on your keyboard, you can get a permalink to that line.